src/client/datascience/notebook/intellisense/intellisenseProvider.ts

```diff
@@ -82,15 +84,16 @@ export class IntellisenseProvider implements INotebookLanguageClientProvider, IE
         const folder =
             this.workspaceService.getWorkspaceFolder(fsPath ? Uri.file(fsPath) : undefined)?.uri ||
             (this.workspaceService.rootPath ? Uri.file(this.workspaceService.rootPath) : undefined);
-        if (folder && !this.activeInterpreterCache.has(folder.fsPath)) {
+        const key = folder ? folder.fsPath : EmptyWorkspaceKey;
```

Comment:
Can't this be an empty string? I'd feel more comfortable with an empty string than using some other hardcoded values.

Reply:
Sure. i don't think it matters.